Client/Frontend/Browser/ZoomPageBar.swift

```diff
@@ -155,6 +163,36 @@ class ZoomPageBar: UIView {
                                           multiplier: UX.separatorHeightMultiplier).isActive = true
     }
 
+    private func setupGradient() {
+        gradient.frame = gradientView.bounds
+        gradient.colors = traitCollection.userInterfaceIdiom == .pad ?
```

Comment:
What is the expectation the gradient on iPad in split view or multitasking? Shouldn't it use the "iPhone" colors then?

Reply:
Basically, the colors are the same, just in reverse ordee, because the Zoom Page bar on the iPad is constrained to the top of the screen.